README.md

```diff
@@ -42,16 +42,33 @@ Resource API documentation is available at https://perseus-demo-energy.ib1.org/a
 
 ## Environment variables
 
-Both apps have example `.env.template` files in their root directories. These should be copied to `.env` and edited as required, filling CLIENT_ID and CLIENT_SECRET with the values provided by Ory Hydra, or on request from ib1 for the demo apps.
+Both apps have example `.env.template` files in their root directories. These should be copied to `.env` and edited as required. The following environment variables are used in the authentication app:
+
+- `REDIS_HOST`: a local redis instance is used to store PAR requests
+- `OAUTH_CLIENT_ID`: Client ID for the Ory Hydra client
+- `OAUTH_URL`: URL for the Ory Hydra client
+- `OAUTH_CLIENT_SECRET`: Client secret for the Ory Hydra client
+- `REDIRECT_URI`: The page to return to after authentication and authorisation eg. for local development http://127.0.0.1:3000/callback
+- `ISSUER_URL`: URL of the Oauth issuer eg. for docker compose https://authentication_web
```

Comment:
I vaguely remember you had some text about how to get these values out of Ory, but can't see it now. Would be nice to refer to it here.